content/en/blog/_posts/2021-12-16-StatefulSet-PVC-Auto-Deletion.md

```diff
@@ -0,0 +1,103 @@
+---
+layout: blog
+title: 'Kubernetes 1.23: StatefulSet PVC Auto-Deletion (alpha)'
+date: 2021-12-16
+slug: kubernetes-1-23-statefulset-pvc-auto-deletion
+---
+
+**Author:** Matthew Cary (Google)
+
+Kubernetes v1.23 introduced a new, alpha-level policy for
+[StatefulSets](docs/concepts/workloads/controllers/statefulset/) that controls the lifetime of
+[PersistentVolumeClaims](docs/concepts/storage/persistent-volumes/) (PVCs) generated from the
+StatefulSet spec template for cases when they should be deleted automatically when the StatefulSet
+is deleted or pods in the StatefulSet are scaled down.
+
+## What problem does this solve?
```

Comment:
should there be a blank line after this title?

Reply:
Markdown (CommonMark) doesn't require a blank line there. See https://deploy-preview-30596--kubernetes-io-main-staging.netlify.app/blog/2021/12/16/kubernetes-1-23-statefulset-pvc-auto-deletion/